app/db/impl/access/ProjectBase.scala

```diff
@@ -149,11 +150,12 @@ class ProjectBase(override val service: ModelService,
 
     channel.remove()
 
+    val p = this.fileManager.getProjectDir(proj.ownerName, proj.name).resolve(channel.name)
```

Comment:
nitpick: please don't use a single character for a variable name

Reply:
that's a temp